nibabel/streamlines/trk.py

```diff
@@ -388,13 +404,13 @@ def _read():
         return cls(tractogram, header=hdr)
 
     def save(self, fileobj):
-        """ Saves tractogram to a file-like object using TRK format.
+        """ Save tractogram to a filename or file-like object using TRK format.
```

Comment:
> I thought the verb was supposed to be conjugated in third-person singular when describing what a function does?


It's not a big deal either way, and I'm inconsistent in doing this,
but I was using the imperative (I think) - there's an example from the
longer PEP document :
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

```
The docstring is a phrase ending in a period. It prescribes the
function or method's effect as a command ("Do this", "Return that"),
not as a description; e.g. don't write "Returns the pathname ...".
```